src/app/organizations/registerOrganization/register-organization.component.html

```diff
@@ -0,0 +1,50 @@
+<!--
+   Copyright 2019 OICR
+ *
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+ *
+       http://www.apache.org/licenses/LICENSE-2.0
+ *
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<h1 mat-dialog-title>Add Organization</h1>
+<div mat-dialog-content>
+  <p>Fill out the form to send an organization request to a Dockstore curator to approve.  Once approved, the organization will be publicly visible.</p>
+  <app-alert></app-alert>
+  <form [formGroup]="registerOrganizationForm">
+    <mat-form-field>
+      <input type="text" matInput formControlName="name" placeholder="Name" required>
+      <mat-error *ngIf="name.hasError('required')">Name is required</mat-error>
+      <mat-error *ngIf="name.hasError('minlength')">Name should be at least 3 characters</mat-error>
+      <mat-error *ngIf="name.hasError('maxlength')">Name should be at most 39 characters</mat-error>
```

Comment:
Need to also check that name starts with a letter.

Reply:
Name can only contain alphanumeric characters and start with an alphabet
->
Name can only contain alphanumeric characters. It must begin with a letter.